useful-scripts/sql/FootnoteData.sql

```diff
@@ -215,11 +213,109 @@ INSERT INTO Footnote (Id, Content, ReleaseId)
 VALUES (@footnote_id_22,
         'Only overall absence is recorded for pupils aged 4, absences are not categorised as authorised or unauthorised.', '4fa4fe8e-9a15-46bb-823f-49bf8e0cdec5');
 
+-- Exclusions footnotes
+DECLARE @footnote_id_23 uniqueidentifier = '16514a6b-4a4c-4200-83c8-c0e3c93a866c';
+DECLARE @footnote_id_24 uniqueidentifier = 'a02f6380-311b-4551-a42e-ba4091879ed8';
+DECLARE @footnote_id_25 uniqueidentifier = 'b5c6814b-78d7-4bf9-b894-92caf334d432';
+DECLARE @footnote_id_26 uniqueidentifier = 'e43f4a88-5cd0-4cad-b84c-d5390f0ecd0f';
+DECLARE @footnote_id_27 uniqueidentifier = '1cf61bb7-b8fc-44ae-a272-e3b2d87d3cf4';
+DECLARE @footnote_id_28 uniqueidentifier = 'a18f2ab2-663a-45cd-8cea-86c716c0453c';
+DECLARE @footnote_id_29 uniqueidentifier = '58717767-3cf5-42b1-8e7a-6c247dfcfc02';
+DECLARE @footnote_id_30 uniqueidentifier = '2b87e363-3a9c-4f43-88c0-e8a602e4c3c0';
+DECLARE @footnote_id_31 uniqueidentifier = '27b8480c-4a53-4385-afb1-69e76f627885';
+DECLARE @footnote_id_32 uniqueidentifier = 'c549699c-83cb-49a3-85a7-6c5e2a91c2b7';
+DECLARE @footnote_id_33 uniqueidentifier = '2eb0b631-48c7-4f4e-b91d-dab25bb9e9a3';
+DECLARE @footnote_id_34 uniqueidentifier = 'eaae1d1f-6e28-4570-b0dd-966366950e60';
+DECLARE @footnote_id_35 uniqueidentifier = '95a70fdf-ea82-4276-869a-22879c5ced20';
+DECLARE @footnote_id_36 uniqueidentifier = '2185b382-8be9-4e48-aba0-096cf5d9b3a3';
+DECLARE @footnote_id_37 uniqueidentifier = '9544977a-24e9-4aee-80d7-77d7cf805986';
+DECLARE @footnote_id_38 uniqueidentifier = 'f2f400ac-b977-4c40-aa34-4a93ef1202f0';
+DECLARE @footnote_id_39 uniqueidentifier = '31457a0b-042c-41a6-9df8-5e4b26e98d6b';
+DECLARE @footnote_id_40 uniqueidentifier = '5c06d319-a0a1-48ed-b747-6a6f8d1354e3';
+DECLARE @footnote_id_41 uniqueidentifier = 'e6890af0-28ae-44c6-8ca9-eaddfb7a381f';
+
+INSERT INTO Footnote (Id, Content, ReleaseId)
+VALUES (@footnote_id_23,
+        'The number of permanent exclusions expressed as a percentage of the number of pupils in January each year.', 'e7774a74-1f62-4b76-b9b5-84f14dac7278');
```

Comment:
It makes even more sense now that there are two Releases to declare two variables with the Release Id and use those variables instead of repeating it. I would have suggested this when the Release Id foreign key got introduced, if I'd spotted it!

Reply:
Done